private/underlay/conn/conn.go

```diff
@@ -321,22 +376,42 @@ func (cc *connUDPBase) initConnUDP(
 	return nil
 }
 
-func (c *connUDPBase) Write(b []byte) (int, error) {
+// @ preserves acc(c.Mem(), _)
+// @ preserves acc(sl.Bytes(b, 0, len(b)), R15)
+// @ preserves unfolding acc(c.Mem(), _) in c.conn == underlyingConn
```

Comment:
I understand that this was done like this in VerifiedSCION, but I think we can improve it to
(1) not mention private fields in the spec
(2) avoid an unnecessary ghost param

To do that, I guess we can have a post condition saying `ensures c.GetUnderlyingConn() == old(c.GetUnderlyingConn())` (we might have to define this function for type `connUDPBase`)